database/seeders/CourseTagSeeder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class CourseTagSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        for ($i = 0; $i < 50; $i++) {
```

Comment:
sao cần for này làm gì em nhỉ?

Reply:
cái này em không tạo model nên em chạy thẳng vòng for trong seeder rồi e vào databaseseeder để gọi cái seeder đó ạ